### PR TITLE
Integrar Multer

### PR DESCRIPTION
**Pull request by LuisPerez007**
.....

### CODE REVIEW

File: src/controller/routes/images-routes.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    const imageRouter = express.Router();
+    //const uploader = uploadMiddleware(1024 * 1024 * 8); // 8MB
+    imageRouter.get('/bucket', (req, res) => {
+        res.status(200).json({ message: 'GET /bucket' });
+        
+    });
+
+    imageRouter.post(
+        '/bucket',
+        //uploader.single('image'), // Middleware para manejar la imagen
```
Comment (Ditmar) [inline src/controller/routes/images-routes.ts]: remove this comment

File: src/controller/routes/images-routes.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        (req, res) => {
+            //console.log(req.file);
+
+            if (!req.file) {
+                return res.status(400).json({ error: 'No se subió ninguna imagen.' });
```
Comment (Ditmar) [inline src/controller/routes/images-routes.ts]: don't hard-coded the code status

File: src/controller/routes/volume-routes.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            } as VolumeModel;
+            const volume = await database.create(volumeData);
+            if (!volume) {
+                return res.status(404).json({
+                    message: 'Volume not created'
+                });
+            }
+            return res.status(201).json({
+                volume,
+                imageMessage: 'Imagen cargada exitosamente.',
```
Comment (Ditmar) [inline src/controller/routes/volume-routes.ts:51]: use ingles for the messages

**Review by Ditmar — CHANGES_REQUESTED** [review]
please add the evidences of your test using postman

**Comment by LuisPerez007** [comment]
PRUEBAS
image
image
image
image
image

**Pull request closed without merging**